src/stdlib.js

```diff
@@ -11,6 +11,27 @@ export const lightscriptImports = {
   "bitwiseZeroFillRightShift": "inline",
 };
 
+export const runtimeHelpers = {
```

Comment:
My current implementation of `hasProps` and `hasLength` doesn't change due to this factor, which I think is desirable, so I think I'll leave this as-is for now, hope that's ok